touchforms/formplayer/tests/test_experiment.py

```diff
@@ -0,0 +1,54 @@
+from django.test import SimpleTestCase
+from touchforms.formplayer.experiments import formplayer_string_compare, formplayer_compare
+
+
+class ExperimentTest(SimpleTestCase):
+
+    control = {}
+    candidate = {}
+
+    def testExperimentStringCompare(self):
+        # session_id will never be equal, should be ignored
+        assert formplayer_string_compare("123", "234", current_key="session_id")
```

Comment:
for tests its preferable to use `self.assertTrue()`, `self.assertFalse`, `self.assertEqual`, etc.